uvicorn/config.py

```diff
@@ -122,7 +122,7 @@ class _IPKind(Enum):
 
 
 def _get_server_start_message(
-    host_ip_version: _IPKind = _IPKind.IPv4,
+    host_ip_version: Optional[_IPKind] = None,
```

Comment:
@euri10 
Looks like I was wrong proposing `_IPKind` enum in https://github.com/encode/uvicorn/pull/803 . Looks like this function has 2 modes: either "to get a message for ipv6" and "to get a message for ipv4/hostname"

What do you think if we delete the `_IPKind` enum and change this function signature to 
```suggestion
    *, is_ipv6_message: bool = False
```
(and also change the function body & usage)

Reply:
yup good idea, I didnt see it coming either :hamburger: